ex.py

```
@@ -0,0 +1,99 @@
+        initial_write_args.extend(["-XMP-iptcExt:PersonInImage+=", person])
+    initial_write_args.append(str(test_image_path))
+    
+    run_exiftool(initial_write_args)
+
+    # --- ÉTAPE 2 : Vérifier les métadonnées initiales ---
+    print("\n--- ÉTAPE 2 : Vérification des métadonnées initiales ---")
+    metadata_after_initial_write = read_metadata(test_image_path)
+    written_people = metadata_after_initial_write.get("XMP-iptcExt", {}).get("PersonInImage")
+    
+    print(f"Personnes lues dans le fichier : {written_people}")
+    assert written_people == initial_people, f"Échec de l'écriture initiale! Attendu: {initial_people}, Obtenu: {written_people}"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)